src/main/java/com/google/sps/servlets/ListOrganizationsServlet.java

```diff
@@ -72,7 +82,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   /* This function constructs a query based on the request parameters & user's role */
   public Query getQueryFromParams(HttpServletRequest request, GivrUser currentUser) {
-    Query query = new Query("Distributor").addSort("creationTimeStampMillis", SortDirection.DESCENDING);;
+    Query query = new Query("Distributor").addSort("creationTimeStampMillis", SortDirection.DESCENDING);
```

Comment:
nit: this and below could have been in another PR.

Reply:
next time I'll be more careful about separating these